src/index.html

```diff
@@ -6,9 +6,44 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="link">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="link__logo">
+        <img class="logo__img" src="./images/logo.png" alt="logo">
```

Comment:
```suggestion
        <img 
          class="logo__img" 
          src="./images/logo.png" 
          alt="logo"
        >
```

Reply:
This should repair npm tests. Please do it for all tags with more than 2 attributes.